### PR TITLE
Update ChallengePanel Flexbox

### PR DESCRIPTION
**Pull request by Sheitak**
Hello,

Response to this issue : #841 

I integrated flex-box, however, it is very difficult for me to see the result without launching the application and visualizing things correctly,

thank you for indicating to me the future modification as clearly as possible if necessary, thank you,

### CODE REVIEW

**Comment by kantord** [comment]
> I integrated flex-box, however, it is very difficult for me to see the result without launching the application and visualizing things correctly,

you can launch the application.

you just need to

```
yarn install
yarn web dev
```

as described in the development documentation: https://librelingo.readthedocs.io/en/latest/

Sorry if that was confusing. Do you think the documentation could be improved in some way?

**Comment by kantord** [comment]
BTW each PR also has it's own stating deployment, see this comment to preview the app with your changes https://github.com/kantord/LibreLingo/pull/867#issuecomment-717953309

**Comment by kantord** [comment]
2020-10-28-204903_1021x804_scrot

the layout seems to be broken

File: workspaces/web/src/components/ChallengePanel.svelte [hunk 1]
```diff
@@ -80,11 +82,8 @@
     color: $text-invert;
     background: $yellow;
   }
-  .left {
-    float: left;
-  }
-  .right {
-    float: right;
+  .container {
```
Comment (kantord) [inline workspaces/web/src/components/ChallengePanel.svelte:85]: this is already defined by bulma https://bulma.io/documentation/layout/container/ 

perhaps you could be able to even just reuse something from bulma

**Review by kantord — CHANGES_REQUESTED** [review]

**Comment by kantord** [comment]
Hi @Sheitak how is it going? are you stuck? do you need some help?

**Comment by Sheitak** [comment]
Hello @kantord , sorry for that response time, I'm busy on other issues right now, struggled because it's not my tech environment, I'm trying to finish this when I have a little time ,

**Comment by kantord** [comment]
Hi @Sheitak  , how is it going? Do you need some help here?

**Comment by kantord** [comment]
Feel free to re-open later if you intend to finish it. Feel free to reach out if you need help!

**Pull request closed without merging**